frontend/src/sagas.js

```diff
@@ -549,17 +557,27 @@ export function* getEnvironmentSaga() {
   });
 }
 
-export function* fetchNotificationsSaga({ showLoading, offset, limit }) {
-  yield commonfetchNotifications(showLoading, offset, limit, FETCH_ALL_NOTIFICATIONS_SUCCESS);
-}
-
-export function* commonfetchNotifications(showLoading, offset, limit, type) {
+export function* fetchNotificationsSaga({ showLoading, notificationPage }) {
   yield execute(function*() {
-    // Get most recent items with negative offset
-    const { data } = yield callApi(api.fetchNotifications, 0 - offset - limit, limit);
+    const { data: notificationCountData } = yield callApi(api.fetchNotificationCounts);
+    const { notificationPageSize } = yield select(getNotificationState);
+
+    const totalNotificationCount = notificationCountData.total;
+
+    const numberOfNotificationPages =
+      notificationPageSize !== 0 ? Math.ceil(totalNotificationCount / notificationPageSize) : 1;
```

Comment:
Not sure if this calculation is needed! We can always go to the first page when changing the page size

Reply:
i think you mean the calculation in another file: https://github.com/openkfw/TruBudget/pull/303/files#diff-0d925ba8dfd401d545b4c938fb855ee6R39

this was removed and the first page is fetched instead.